mediator/test/workflows.spec.ts

```diff
@@ -160,24 +160,24 @@ describe('Workflows', () => {
 
     });
 
-    it('Should follow the OpenMRS Patient to CHT workflow', async () => {
+    it.skip('Should follow the OpenMRS Patient to CHT workflow', async () => {
```

Comment:
Could you please add a comment explaining why it's skipped and when you plan to re-enable it? This will help others understand the context and know when to revisit it.

Reply:
eneded up unskipping it and fixing it. There are some TODOs left that could be added, but for now it tests getting a patient by id after saving it which is enough to keep it running